bindings/sampling-heap-profiler.cc

```diff
@@ -81,13 +81,26 @@ NAN_METHOD(GetAllocationProfile) {
   info.GetReturnValue().Set(TranslateAllocationProfile(root));
 }
 
+NAN_METHOD(GetAllocationProfileProto) {
+  if (info.Length() != 1 || !info[0]->IsNumber()) {
+    return Nan::ThrowTypeError("First argument type must be Integer.");
+  }
+  int64_t startTimeNanos = info[0].As<Integer>()->IntegerValue();
+  std::unique_ptr<v8::AllocationProfile> profile(
+    info.GetIsolate()->GetHeapProfiler()->GetAllocationProfile());
+  std::vector<char>b = serializeHeapProfile(std::move(profile), 512 * 1024, startTimeNanos);
+  info.GetReturnValue().Set(Nan::CopyBuffer( &b[0], b.size()).ToLocalChecked());
```

Comment:
Remove space before "&b[0]". Can you run the code through clang-format with something like https://github.com/grpc/grpc/blob/5d0b1aeadecf5cbfafa371da9494cd600615733a/tools/distrib/clang_format_code.sh?

Reply:
Done.
Have used clang-format now.